installing/installing_nutanix/nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nutanix-failure-domains"]
+= Fault tolerant deployments using multiple Prism Elements
+include::_attributes/common-attributes.adoc[]
+:context: nutanix-failure-domains
+
+toc::[]
+
+By default, the installation program installs control plane and compute machines into a single Nutanix Prism Element (cluster). To improve the fault tolerance of your {product-title} cluster, you can specify that these machines be distributed across multiple Nutanix clusters by configuring failure domains.
+
+A failure domain represents an additional Prism Element instance that is available to {product-title} machine pools during and after installation.
+
+include::modules/installation-nutanix-failure-domains-req.adoc[leveloffset=+1]
+
+== Installation method and failure domain configuration
+
+The {product-title} installation method determines how and when you configure failure domains:
+
+* If you deploy using installer-provisioned infrastructure, you can configure failure domains in the installation configuration file before deploying the cluster. For more information, see xref:../../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuring-nutanix-failure-domains_installing-nutanix-installer-provisioned[Configuring failure domains].
++
+You can also configure failure domains after the cluster is deployed by modifying the cluster's Infrastructure custom resource definition (CRD) (`infrastructures.config.openshift.io`).
+* If you deploy using the Assisted Installer or to infrastructure that you manage (user-provisioned infrastructure), you configure failure domains after the cluster is deployed by modifying the cluster's Infrastructure CRD.
```

Comment:
Also need to modify the ControlPlaneMachineSet resource if the control-plane nodes need to spread through the newly added Failure Domains.